x/bridge/observer/osmosis/client_test.go

```diff
@@ -152,7 +179,6 @@ func TestSignTxSuccess(t *testing.T) {
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
 	)
 	require.NoError(t, err)
-	defer conn.Close()
 
 	keyring := keyring.NewInMemory(app.GetEncodingConfig().Marshaler)
```

Comment:
variable name collides with the package name
```suggestion
	kr := keyring.NewInMemory(app.GetEncodingConfig().Marshaler)
```